kin-sdk/src/main/java/kin/sdk/WhitelistableTransaction.java

```diff
@@ -0,0 +1,21 @@
+package kin.sdk;
+
+public class WhitelistableTransaction {
+
+    private final String transactionEnvelopXdrBase64;
+    private final String networkPassphrase;
+
+    public WhitelistableTransaction(String transactionEnvelopXdrBase64, String networkPassphrase) {
+        this.transactionEnvelopXdrBase64 = transactionEnvelopXdrBase64;
+        this.networkPassphrase = networkPassphrase;
+    }
+
+    public String getTransactionEnvelopXdrBase64() {
```

Comment:
seems very detailed, maybe `TransactionPayload`?
@Ronserruya @cnotethegr8 @drEfrati please sync on the terminology here across platforms.

Reply:
@drEfrati sync with Corey on the naming here please